Removed linkage.h from glibc, it is not part of the exported 
kernel header files any more.

--- libc/sysdeps/unix/sysv/linux/st200/syscall.S.orig	2008-05-20 13:20:01.000000000 +0100
+++ libc/sysdeps/unix/sysv/linux/st200/syscall.S	2008-05-20 13:20:10.000000000 +0100
@@ -17,7 +17,6 @@
    02111-1307 USA.  */
 
 #include <sysdep.h>
-#include <linux/linkage.h>	
 #include <sys/syscall.h>
 			
 	.text
--- libc/sysdeps/unix/sysv/linux/st200/clone.S.orig	2008-05-20 13:33:54.000000000 +0100
+++ libc/sysdeps/unix/sysv/linux/st200/clone.S	2008-05-20 13:45:43.000000000 +0100
@@ -25,7 +25,6 @@
    stack to store them. */
 
 #include <sysdep.h>
-#include <linux/linkage.h>
 #define _ERRNO_H        1
 #include <bits/errno.h>
 #include <asm/unistd.h>
@@ -33,7 +32,10 @@
 // int __clone(int (*fn)(void *), void *child_stack, int flags, void *arg,
 //             pid_t *ptid, void *tls, pid_t *ctid)
 
-ENTRY (__clone)
+	.text
+	.proc
+	.global C_SYMBOL_NAME(__clone)
+C_LABEL(__clone)
 	andl    $b0 = $r16, $r17   /* Test for NULL fn ptr or NULL stack ptr */
 	;;
 	brf	$b0, L?error_einval
@@ -95,6 +97,6 @@
 	SYSCALL_ERROR_HANDLER
 	mov $r16 = -1
 	ret
-END(__clone)
+	.endp
 
 weak_alias(__clone, clone)
